README.md

```diff
@@ -57,4 +57,7 @@ You should put the .jar in an extra folder as a *logs* and a *db* folder will be
 
 * Windows 7, 10
 * Linux (tested on Ubuntu 18.04)
```

Comment:
can you also test the .jar file on windows + linux. When tested also update the readme which Operating systems we tested (Windows 10 + Ubuntu version x)

Reply:
Works on Windows 10

Reply:
on linux only import/export is not working. see https://github.com/doubleSlashde/KeepTime/pull/109